scripts/FONTS.md

```diff
@@ -0,0 +1,8 @@
+The Americana fontstack is built from [Google Fonts](https://fonts.google.com/).
+
+In order to add fontstack support, modify the `fonts.json` file as follows:
+
+1. Add font-family and variant information to the `font-families` section. The font-family is the name of the font as listed on Google Fonts, e.g. "Noto Sans". Use `gfi download "<name of font>"` to get a full list of the variants. The variant is everything to the right of the dash in the filename, so if a file is named `NotoSans-700.ttf`, the variant is `700`, though it will be listed in Google as something like "Bold 700". The `gfi` command requires you to install the `google-font-installer` package into npm with `npm install -g google-font-installer`.
```

Comment:
Not sure if it should go here or in CONTRIBUTING.md, but we could document some editorial guidelines to aid in choosing fonts in non-Western writing systems. For example, once we get around to implementing custom CJK fonts, either on the server side via a fontstack or (preferably) on the client side via standard Web fonts, we’d want to pair Western regular sans-serif with [gothic](https://en.wikipedia.org/wiki/East_Asian_Gothic_typeface) and Western regular serif with [Ming](https://en.wikipedia.org/wiki/Ming_%28typefaces%29), and Western italic sans-serif with [regular](https://en.wikipedia.org/wiki/Regular_script#In_computing), based on what Western and CJK texts do in analogous situations.

That said, one of the outgrowths of this PR may end up being a separate project specifically about fonts, where we’d naturally explore and elaborate on these ideas.

Reply:
Tracking Unicode ligatures and combining characters in #827